test/TensorFlowNET.UnitTest/Eager/CApi.Eager.cs

```diff
@@ -17,25 +17,23 @@ IntPtr TestMatrixTensorHandle()
             var t = c_api.TF_AllocateTensor(TF_FLOAT, dims, dims.Length, (ulong)data.Length * sizeof(float));
             memcpy(c_api.TF_TensorData(t), data, data.Length * sizeof(float));
             
-            var status = c_api.TF_NewStatus();
+            using var status = c_api.TF_NewStatus();
             var th = c_api.TFE_NewTensorHandle(t, status);
             CHECK_EQ(TF_OK, TF_GetCode(status), TF_Message(status));
             c_api.TF_DeleteTensor(t);
-            c_api.TF_DeleteStatus(status);
             return th;
         }
 
         IntPtr MatMulOp(IntPtr ctx, IntPtr a, IntPtr b)
         {
-            var status = TF_NewStatus();
+            using var status = TF_NewStatus();
 
             var op = TFE_NewOp(ctx, "MatMul", status);
             CHECK_EQ(TF_OK, TF_GetCode(status), TF_Message(status));
             TFE_OpAddInput(op, a, status);
             CHECK_EQ(TF_OK, TF_GetCode(status), TF_Message(status));
             TFE_OpAddInput(op, b, status);
             CHECK_EQ(TF_OK, TF_GetCode(status), TF_Message(status));
-            TF_DeleteStatus(status);
             TFE_OpSetAttrType(op, "T", TFE_TensorHandleDataType(a));
```

Comment:
📝 This one changed the order of operations, but it seemed acceptable for two reasons:

1. It didn't seem to matter which order these two lines appear in, as the second does not use `status`
1. This is test code, so it's not risking a subtle change to shipping code to switch the order here

A few other tests in the same file have the same situation.